recipes/mcxtrace-suite/install-files-core.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+#Make sure we fail hard in case of trouble:
+set -e
+set -u
+set -x
+
+SRCDIR="$PWD/src"
+
+#A few (silent) sanity checks that variables are set and meaningful:
+test -d "${PREFIX}"
+test -d ${SRCDIR}
+test -f ${SRCDIR}/CMakeLists.txt
+test -n "${PKG_VERSION}"
+
+for i in $(seq 1 100000); do
+    #Find a unique build dir:
+    BLDDIR="$PWD/build_mcxtrace_core_${i}"
+    if [ ! -d "${BLDDIR}" ]; then
+        break
+    fi
+done
```

Comment:
I'm not an expert but there must be a safer way to implement this? Also, in our CIs this should not be needed b/c there shouldn't be an existing directory.

Reply:
This is not an issue per se but a brute force approach that we probably don't need.

Reply:
Then I propose we simply remove this stuff unless @tkittel violently objects?

Reply:
I don't object, but it was added only because it turned out that occasionally the build dir was already there (for, to me, mysterious reasons).

Reply:
Then I will take it out - and we add it back in + add a `conda-forge` issue if it reappars.

Reply:
Resolved in https://github.com/conda-forge/staged-recipes/pull/25503/commits/354f62b4cbb118bc4874072343ee6b21032a42f1